README.md

```diff
@@ -11,15 +11,24 @@ make sure that the firmware in your module is compatible with Mbed OS, follow th
 It is advisable to update the [AT firmware](https://www.espressif.com/en/support/download/at?keys=) at least to version
 1.7.0.0
 
+v1.6 driver supports AT firmware up to version 1.6.2. Mbed OS 5.10 supports AT firmwares up to version 1.6.2. v1.7
+driver found from Mbed OS 5.11 allows usage of AT firmware version 1.7.0.0.
+
+**NOTE** Here we are talking about AT firmware version - not SDK version. SDK firmware without AT is no use.
```

Comment:
Is there any coupling of SDK versions and AT firmware versions?

For example, can we simplify this sections so that `AT firmware version v1.7 is released in ESP8266 SDK version v3.x`

Reply:
I don't know. But I'll drop everything what mentions SDK